.gitignore

```diff
@@ -10,3 +10,4 @@ Gemfile*.lock
 .rvmrc
 /doc/
 .ruby-version
+.ruby-gemset
```

Comment:
This can be removed. Instead of adding this to the project's .gitignore file, you can add to a global .gitignore that's local to your workstation. See https://help.github.com/articles/ignoring-files/#create-a-global-gitignore for the details.